pymatgen/io/vasp/MatPESStaticSet.yaml

```diff
@@ -0,0 +1,160 @@
+# Default VASP settings for static calculations in the MatPES Project
+# using the Regularized-Restored Strongly Constrained and Appropriately
+# Normed functional (PBE, PBE+U or r2SCAN).
+INCAR:
+  ALGO: ALL
+  EDIFF: 1.e-05
+  EDIFFG: -0.02
+  ENAUG: 1360
+  ENCUT: 680
+  IBRION: 2
+  ISIF: 3
+  ISMEAR: 0 # change from 2 to 0, included to have some reasonable default
+  ISPIN: 2
+  KSPACING: 0.22 # included to have some reasonable default
+  LAECHG: True
+  LASPH: True
+  LCHARG: True
+  LELF: False # LELF = True restricts calculation to KPAR = 1
+  LMIXTAU: True
+  LORBIT: 11
+  LREAL: Auto
+  LVTOT: True
+  LWAVE: False
+  METAGGA: PBE # change R2SCAN to PBE as default
+  NELM: 200
+  NSW: 0 # static calculation
+  PREC: Accurate
+  SIGMA: 0.05 # change from 0.02 to 0.05, included to have some reasonable default
+  MAGMOM:
+    Ce: 5
+    Ce3+: 1
+    Co: 0.6
+    Co3+: 0.6
+    Co4+: 1
+    Cr: 5
+    Dy3+: 5
+    Er3+: 3
+    Eu: 10
+    Eu2+: 7
+    Eu3+: 6
+    Fe: 5
+    Gd3+: 7
+    Ho3+: 4
+    La3+: 0.6
+    Lu3+: 0.6
+    Mn: 5
+    Mn3+: 4
+    Mn4+: 3
+    Mo: 5
+    Nd3+: 3
+    Ni: 5
+    Pm3+: 4
+    Pr3+: 2
+    Sm3+: 5
+    Tb3+: 6
+    Tm3+: 2
+    V: 5
+    W: 5
+    Yb3+: 1
+POTCAR_FUNCTIONAL: PBE_54
+POTCAR:
```

Comment:
I don't think we should be hard coding POTCARs or U values in this input set. The VASP input sets support inheritance. E.g., VASPIncarBase defines U values. We should write a base for PBE54 that incorporates things we feel would be common across all calculations. E.g., POTCAR_FUNCTIONAL, POTCAR and maybe the U values.

Reply:
I felt confused about how we make sure all the MatPES static calculations have the same PSP, so that's why I incorporate the POTCAR values here. When you mentioned write a base for PBE54, do you mean to include another` PBE54Base.yaml` used for all the PBE54 POTCARs, POTCAR_FUNCTIONAL, and U values?

Reply:
Yes.